ticdc/ticdc-changefeed-config.md

```diff
@@ -131,4 +164,34 @@ quote = '"'
 null = '\N'
 # Whether to include commit-ts in CSV rows. The default value is false.
 include-commit-ts = false
+
+# Represents the replication consistency config for a changefeed when using the redo log, for more information, please refer to https://docs.pingcap.com/tidb/stable/ticdc-sink-to-mysql#eventually-consistent-replication-in-disaster-scenarios. Note: Ignore the following configurations if downstream is MQ or Storage.
+[sink.consistent]
+# The data consistency level, available options are `none` and `eventual`, none means the redo log is disabled.  
+# The default value is none.
+level = "none"
+# The max redo log size in megabytes.
+# The default value is 64  
+max-log-size = 64
+# The flush interval for redo log. Default value is 2000 milliseconds.
+# The default value is 2000 
+flush-interval = 2000 
+# The storage URI of the redo log.
+# The default value is empty
+storage = "" 
+# Specifies whether to store the redo log in a file.
+# The default value is false
+use-file-backend = false
+
+# Specifies the data integrity check. Note: ignore the following configurations if the downstream type is DB or Storage.
```

Comment:
Does “DB” mean “TiDB”？

Reply:
No. DB means MySQL compatible database, it is an internal data type.

Reply:
Has changed to Database